regression-test/suites/external_table_p0/hive/test_multi_delimit_serde.groovy

```diff
@@ -0,0 +1,136 @@
+                CREATE TABLE IF NOT EXISTS multi_delimit_test1 (
+                    k1 int,
+                    k2 int,
+                    name string
+                )
+                ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.MultiDelimitSerDe'
+                WITH SERDEPROPERTIES (
+                    'field.delim' = '|+|',
+                    'mapkey.delim' = '@',
+                    'collection.delim' = ':',
+                    'serialization.format' = '1',
+                    'serialization.encoding' = 'UTF-8'
+                )
+                STORED AS INPUTFORMAT
+                    'org.apache.hadoop.mapred.TextInputFormat'
+                OUTPUTFORMAT
+                    'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+            """
+
+            hive_docker """INSERT OVERWRITE TABLE multi_delimit_test1 VALUES (1, 100, 'test1'), (2, 200, 'test2'), (3, 300, 'test3')"""
+
+            qt_01 """SELECT * FROM multi_delimit_test1 ORDER BY k1"""
+
+            // Test 2: Different multi-character delimiter
+            logger.info("Test 2: Creating table with || delimiter")
+            hive_docker """
+                CREATE TABLE IF NOT EXISTS multi_delimit_test2 (
+                    id int,
+                    value double,
+                    description string
+                )
+                ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.MultiDelimitSerDe'
+                WITH SERDEPROPERTIES (
+                    'field.delim' = '||',
+                    'serialization.format' = '1'
+                )
+                STORED AS INPUTFORMAT
+                    'org.apache.hadoop.mapred.TextInputFormat'
+                OUTPUTFORMAT
+                    'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+            """
+
+            hive_docker """INSERT OVERWRITE TABLE multi_delimit_test2 VALUES (1, 1.5, 'desc1'), (2, 2.5, 'desc2')"""
+
+            qt_02 """SELECT * FROM multi_delimit_test2 ORDER BY id"""
+
+            // Test 3: Complex multi-character delimiter
+            logger.info("Test 3: Creating table with complex delimiter")
+            hive_docker """
+                CREATE TABLE IF NOT EXISTS multi_delimit_test3 (
+                    col1 string,
+                    col2 string,
+                    col3 string
+                )
+                ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.MultiDelimitSerDe'
+                WITH SERDEPROPERTIES (
+                    'field.delim' = ':::'
+                )
+                STORED AS INPUTFORMAT
+                    'org.apache.hadoop.mapred.TextInputFormat'
+                OUTPUTFORMAT
+                    'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+            """
+
+            hive_docker """INSERT OVERWRITE TABLE multi_delimit_test3 VALUES ('field1', 'field2', 'field3'), ('a', 'b', 'c')"""
+
+            qt_03 """SELECT * FROM multi_delimit_test3 ORDER BY col1"""
```

Comment:
Please also test the insert command (Use Doris to write to Hive) for each table type